tournaments/round_robin_tournament.py

```diff
@@ -0,0 +1,51 @@
+from core.connect_four_game import ConnectFourGame
+from core.board import Board
+from itertools import permutations
+from functools import reduce
+
+class RoundRobinTournament:
+    def __init__(self, players, view):
+        self.players = players
+        self.matches = []
+        self.results = {}
+        self.view = view
+    
+    def _calc_match_permutations(self):
```

Comment:
is this the convention for a private method?

Reply:
Kind of, a single _ is meant to be protected and double _ is meant.to be private. But generally an _ don't touch this outside of this class.